WBooks/Components/Suggestions/cell/SuggestionsCollectionViewCell.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  SuggestionsCollectionViewCell.swift
+//  WBooks
+//
+//  Created by daniel.f.munoz on 9/09/21.
+//
+
+import UIKit
+
+class SuggestionsCollectionViewCell: UICollectionViewCell {
+    
+    @IBOutlet weak var suggestBookImage: UIImageView! {
+        didSet {
+            suggestBookImage.contentMode = .scaleAspectFit
+        }
+    }
+    
+    static let identifier = "SuggestionsCollectionViewCell"
+    static let nibName = "SuggestionsCollectionViewCell"
```

Comment:
What is it the diference between identifier and nibName?

Reply:
The identifier refers to the cell, and the nibName refers to the name of the .xib file, in this case both are called the same, but leave them independently in case you need to change the name in the future this does not affect where they are currently called.